zhmccli/_helper.py

```diff
@@ -224,7 +224,7 @@ class CmdContext(object):
 
     def __init__(self, host, userid, password, no_verify, ca_certs,
                  output_format, transpose, error_format, timestats, session_id,
-                 get_password):
+                 get_password, pdb):
```

Comment:
Yeah, looks like you have the same commit in 2 PRs?

Reply:
That was intentional. See previous response.